.github/workflows/sauce.yml

```
@@ -0,0 +1,24 @@
+name: Sauce Labs
+on: push
```

```suggestion
on: push
concurrency: sauce-labs
```

See https://github.com/Level/level-js/pull/212